Client.py

```diff
@@ -62,13 +65,15 @@ def run(self):
             self.update()
             self.player.move()
             self.drawPlayer()
+            if self.player.newBomb:
+                connection.Send({"action": "newBombFromPlayer", "bomb": self.player.bombToDictionary(self.player.newBomb)})
+                self.player.newBomb = 0
```

Comment:
This should be extracted to a seperate function

Reply:
Done